app/models/concerns/hyrax/work_type_icon.rb

```diff
@@ -0,0 +1,25 @@
+module Hyrax
+  module WorkTypeIcon
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      def work_type_icon
+        @_work_type_icon || Hyrax.config.default_work_icon
+      end
+
+      def work_type_icon=(val)
+        @_work_type_icon = val
+      end
+    end
+
+    def work_type_icon
+      self.class.work_type_icon
+    end
+
+    def to_solr(solr_doc = {})
+      super(solr_doc).tap do |doc|
+        doc[Solrizer.solr_name('work_type_icon')] = work_type_icon
```

Comment:
Why does this need to be indexed? Should it it just be config? Should we have to reindeer everything if we wanted to change it?

Reply:
It may not need to be. I was following the human_readable_type example, but can test this without indexing. I'll follow up.

Reply:
The human readable type is required because it's used (or was at it's addition)  as a facet.

Reply:
@jcoyne just realized I should be able to pull this from the i18n config. Doh! Will refactor with that in mind.

Reply:
Please don't use i18n for this, because you'd have to copy the value to every language (and it's unlikey to change based on language). Just the Hyrax::Configuration should be fine.

Reply:
It's generated into the work_name.lang.yml file though. Seems like the best place to grab/set it..